src/components/PoolDetail.tsx

```
@@ -0,0 +1,92 @@
+"use client";
+import { useEffect, useState } from "react";
+import { Tabs, TabsContent, TabsList, TabsTrigger } from "@/components/ui/tabs";
+import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
+import { Button } from "@/components/ui/button";
+import { fetchPool, fetchUserInfo } from "@/lib/blend/helpers";
+
+export function PoolDetail({ id }: { id: string }) {
+  const [loading, setLoading] = useState(true);
+  const [pool, setPool] = useState<any>(null);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)